runtime/testy/studentAPI.py

```diff
@@ -2,20 +2,20 @@
 
 class Robot:
   def __init__(self, toManager, fromManager):
-    self.fromManager = fromManager
-    self.toManager = toManager
+    self.fromManager = fromManager #pipe
+    self.toManager = toManager #statequeue
     # Wait for ack from SM before continuing
     self.fromManager.recv()
 
-  def getValue(self, name):
-    """Returns the value associated with name
+  def getValue(self, key):
+    """Returns the value associated with key
     """
-    # TODO: Actually use name
-    self.toManager.put([SM_COMMANDS.HELLO])
+    # TODO: Actually use key
+    self.toManager.put([SM_COMMANDS.GET_VAL, key])
     return self.fromManager.recv()
 
-  def setValue(self, name, value):
-    """Sets the value associated with name
+  def setValue(self, key, value):
+    """Sets the value associated with key
     """
     # TODO: Implement
-    raise NotImplementedError
+    self.toManager.put([SM_COMMANDS.SET_VAL, key, value])
```

Comment:
I think new line is there. Github hid it because it is the same in both files. 
I think it'll be red text if its missing a new line